src/components/ec-blog-item/ec-blog-item.story.js

```diff
@@ -0,0 +1,41 @@
+import { storiesOf } from '@storybook/vue';
+import EcBlogItem from './ec-blog-item.vue';
+
+const stories = storiesOf('Blog Item', module);
+
+stories.add('all', () => ({
+  components: { EcBlogItem },
+  data() {
+    return {
+          id: 1,
+          title: 'Post 1',
+          shortDescription: 'Short 1',
+          author: 'Author 1',
+          category: 'Category 1',
+    };
+  },
+  props: {
+    title: {
+      default: text('title', 'A new title'),
```

Comment:
When I try to see the storybook link I'm getting that text is not defined
![image](https://user-images.githubusercontent.com/31845091/99783195-6a730900-2b1a-11eb-8022-d1e3805243db.png)